code/modules/awaymissions/zlevel.dm

```diff
@@ -6,7 +6,13 @@
 		air_master.setup_allturfs(turfs)
 	for(var/turf/T in turfs)
 		if(T.dynamic_lighting)
-			T.lighting_build_overlays()
+			T.lighting_build_overlay()
+		T.lighting_corners_initialised = TRUE
```

Comment:
#4780 includes porting an updated map loader that uses ChangeTurf... though it seems to still have some unresolved issues.